backend/routes/curriculums.js

```diff
@@ -89,4 +89,12 @@ router.post(
   })
 );
 
+router.get(
+  "/getCurriculumYears",
+  asyncMiddleware(async function(req, res, next) {
```

Comment:
Just a question, does asyncMiddleware need async? is it redundant? Or do you need to specify async?